main.tf

```diff
@@ -19,7 +19,7 @@ data "google_project" "project" {
 }
 
 locals {
-  api_image = "gcr.io/sic-container-repo/todo-api-postgres:latest"
+  api_image = (var.database_type == "mysql" ? "gcr.io/sic-container-repo/todo-api" : "gcr.io/sic-container-repo/todo-api-postgres:latest")
```

Comment:
**Praise:** Thanks for digging into the differences between these two images! :) I didn't realize to original 3-tier web app used MySQL.